app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    integer = split_integer(8, 2)
+    assert sum(integer) == 8
```

Comment:
```suggestion
    assert sum(split_integer(8, 2)) == 8
```
read checklist.md
![image](https://github.com/user-attachments/assets/42fb0ac1-4b77-4335-b051-000dd78061e9)

Reply:
@vsmutok So why AI approve these tasks?

Reply:
@KostiantinPastukh  because it's AI😅, it can make mistakes sometimes